u19_pipeline/ephys_pipeline.py

```diff
@@ -35,62 +34,42 @@
 ephys_schema_name = dj.config['custom']['database.prefix'] + 'ephys_element'
 
 # 2. Upstream tables
-schema_reference = dj.schema(dj.config['custom']['database.prefix'] + 'reference')
-schema = dj.schema(dj.config['custom']['database.prefix'] + 'ephys')
+reference_schema = dj.schema(dj.config['custom']['database.prefix'] + 'reference')
 
-
-@schema_reference
+@reference_schema
 class SkullReference(dj.Lookup):
     definition = """
     skull_reference   : varchar(60)
     """
     contents = zip(['Bregma', 'Lambda'])
 
-
-@schema
-class EphysSession(dj.Manual):
-    definition = """
-    # General information of an ephys session
-    -> acquisition.Session
-    ---
-    ephys_directory: varchar(255)      # the absolute directory where the ephys data for this session will be stored in bucket
-    """
-
-
-# ephys element requires table with name Session
-Session = EphysSession
-
+Session = recording.EphysRecordingSession
```

Comment:
**EphysRecordingSession** will be in ephys_pipeline "pre_element" schema right ?

Reply:
Added `EphysRecordingSession` to the `ephys_recording` module/schema, which is upstream from `ephys_pipeline`.